src/components/Login/Login.js

```diff
@@ -37,9 +37,6 @@ export default function Login({goBack}) {
 }
 
 const styles = StyleSheet.create({
-    container:{
-        flex:1,
-    },
     containerLogin: {
         backgroundColor: '#000',
```

Comment:
May be it's better to create separate config file with style constants: colors (primary, secondary, light, dark), font sizes (like sm, md, lg) and use it through whole project. Also it's great to read about color schemes theory. It's really useful to undestand why you need three/four colors through project (primary, secondary, tertiary).